service/pixelated/resources/root_resource.py

```diff
@@ -40,10 +40,10 @@
 
 
 class RootResource(BaseResource):
-    def __init__(self, services_factory):
+    def __init__(self, services_factory, static_folder=None):
```

Comment:
I agree the `_get_static_folder()` looks a bit hacky.
Maybe we could remove it and add the default at `arguments.py`, like: `RootResource(services_factory, args.static_folder)`